lxd/network/driver_ovn.go

```diff
@@ -3935,6 +3935,26 @@ func (n *ovn) InstanceDevicePortDynamicIPs(instanceUUID string, deviceName strin
 	return client.LogicalSwitchPortDynamicIPs(instancePortName)
 }
 
+// InstanceDevicePortIPs returns the allocated IPs for a device port.
+func (n *ovn) InstanceDevicePortIPs(instanceUUID string, deviceName string) ([]net.IP, error) {
+	if instanceUUID == "" {
+		return nil, fmt.Errorf("Instance UUID is required")
+	}
+
+	client, err := openvswitch.NewOVN(n.state)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to get OVN client: %w", err)
+	}
+
+	switchPortIPs, err := client.LogicalSwitchPortIPs(n.getIntSwitchName())
```

Comment:
Can we modify  client.LogicalSwitchPortIPs to accept option (if non-empty) logical switch port name that it would filter by in the ovn query, to save pulling all ip info back from ovn?

Reply:
Makes sense to me!